buglist-link.js

```diff
@@ -0,0 +1,37 @@
+var bugs = [];
+
+function getCommonLandings() {
+	//look for Bug # on page
+	let result;
+	let re = /Bug ([0-9]+)/ig;
+	while ((result = re.exec(document.body.innerText)) !== null) {
+		bugs.push(result[1]);
+	}
+	//consolidate Bug #
+	bugs = new Set(bugs);
```

Comment:
bugs should be a Set instead of a list at definition, and use add instead of push